cropper/app/controllers/CropperController.scala

```diff
@@ -79,7 +81,7 @@ class CropperController(auth: Authentication, crops: Crops, store: CropStore, no
     }
   }
 
-  private val canDeleteCrops: PrincipalFilter = authorisation.hasPermissionTo(DeleteCrops)
+  private val canDeleteCrops: PrincipalFilter = authorisation.hasPermissionTo(DeleteCropsOrUsages)
```

Comment:
does this also want to say `OrUsages`?

Reply:
Didn't see the need here since it's a value in Cropper service and only used for deleting crops. But could rename as well if you think it's appropriate.